test/data/config.yaml

```diff
@@ -33,4 +33,4 @@ packages:
   directory: "/mnt/directory-with-rpm-files"
   directory: "/mnt/another-directory-with-rpm-files"
 
-call: "/mnt/foo/update.sh"
+call: "foo/update.sh"
```

Comment:
I don't think we want to enforce that this is a shell script. The only requirement we should have is that this file has the executable bit set. Which will work based on the implementation of the call with "bash -c"

Reply:
right I do not enforce this to be a shell script. As you can see this is only the test data file where I just picked up a name. It could be anything. What is your point ? should we pick a more generic name which doesn't lead to the impression it must be a shell script ? if so I could update the docstring of the runtime_config source which holds an example config but the unit test data is imho irrelevant and also not populated to users

Reply:
Well when people are uncertain they might look at the test and then get the impression it has to be a shell script, so yes, a more explicit doc string with "Run file with executable permission set" is probably advisable

Reply:
ok done